### DISCUSSION TITLE
Are we turning into a pay to play?

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1405] [General]
I have been playing this game non-stop for many months, always invested, never withdrawn a coin. Is the entertainment of this game spending money? I can't achieve anything, I can't have a strategy. I thought the cake event would unburden my farm, but no, I had to invest. Now comes the war, high resources, which we don't have, are required. NTFS with low supply, what does it mean? Have to invest! My farm is not sustainable and it seems it never will be... It seems that the game is being thought by people who have all the boosts. If it's hard for me who's been playing for a long time, imagine who's starting now? How people that starting today participate of War? Don't you think this can bring frustration to these players? How does someone recommend such a game to a friend? You saw how long Apron, that was for a group, was in stock and now launch amulets, that are for all, with this stock?

Little ones are getting crushed in this game. This is not interesting even for the big ones, who won't have anyone to sell their wares to.

**Comment by adamhannigan** [comment]
@future2back Thanks for raising your concern. The team is aware of this issue and is the main reason why we have been focussing on land expansion.

The game model as it stands does not support millions of users. When we launch we aim to have gameplay styles that accomodate all types of players

**Reply by future2back** [reply]
@adamhannigan  thanks for replying.
Meanwhile what must we do? You should take action as soon as possible. I feel like I didn't gain anything by playing the beta version. All the updates never harmed this select group and the team of developers while normal players are forgotten.
This event is a temporary event, it should be reachable for everyone who is participating, but again it is reachable for the select group and sync encourager.

So who is the stronger in this war? The best player or best payer?

5k+ war bounds in 1 week is it normal?

We are very close to a halvening is the focus making the little ones give up?

Think about it.

**Reply by future2back** [reply]
If you thought that:
500 potatoes
50 carrots
50 cauliflower
It was very expensive... Imagine this week. _10 axes + 5 radish for 10 war bounds_.

This is a punch in the face for those who still believed in changes.

Congratulations to the development team, if the goal is to discourage small and new players, you are on the right way.

**Reply by adamhannigan** [reply]
@future2back Apologies, I missed these comments. 

The beta is definitely flawed, and part of being in early testing. 

We had a few options when it came to the timeline:

- Make immediate fixes (6-8 weeks work). This would delay our launch by 2 months but give short term relief to players
- Make long term fixes. Resolve the game issues in the longer term but cause short term boredom

In this case we took the option that ensured the longevity of the game. 

Whilst doing this we still managed to release multiple features to help with engagement, but it would never solve the issue. We are extremely grateful for the players who have stuck with us during this period of Beta testing.
